web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -316,14 +307,16 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
         return chunks.join("-");
     },
 
-    _viewButtonOnClick: function(btn) {
+    _viewButtonOnClick: function(btn, clickEvent) {
 
-        btn.addClass("x-btn-selected");
+        var multiSelect = clickEvent.ctrlKey;
         var superUuid = btn.container.id.replace("fsSearchAddBtn", '');
-        this.addRecordFromSuperUuid(superUuid);
+        this.addRecordFromSuperUuid(superUuid, multiSelect);
+
+        btn.addClass("x-btn-selected");
     },
 
-    addRecordFromSuperUuid: function(superUuid) {
+    addRecordFromSuperUuid: function(superUuid, multiSelect) {
```

Comment:
It was exposed to a neutron blast when it was a young UUID and now it has great power (and responsibility) and is a *super*UUID.

Also, from what I can tell there were cases where the same metadata records were being returned in the search results. So although their UUIDs were unique for each one, they weren't unique on the page. So the superUUID combines the row number to make them unique on the page.

I don't know why there would be duplicates in the search results. Perhaps that is a bug which has since been fixed. I will check with @pmbohm who I think made those changes.